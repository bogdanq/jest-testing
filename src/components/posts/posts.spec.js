import React from "react";
import { Posts } from "./index";

describe("Posts component", () => {
  it("should shallow Posts component", () => {
    const component = shallow(<Posts />);
    expect(component).toMatchSnapshot();
  });

  it("should render Posts component", () => {
    const component = render(<Posts />);
    expect(component).toMatchSnapshot();
  });
});
