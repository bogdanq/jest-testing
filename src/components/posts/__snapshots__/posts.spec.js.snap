// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Posts component should render Posts component 1`] = `
Array [
  <div
    class="post"
  >
    <p>
      12
    </p>
    <span
      class="date"
    >
      12.12.2020
    </span>
    <a
      href="12"
    >
      12
    </a>
  </div>,
  <div
    class="post"
  >
    <p>
      12
    </p>
    <span
      class="date"
    >
      22.12.2020
    </span>
    <a
      href="12"
    >
      12
    </a>
  </div>,
]
`;

exports[`Posts component should shallow Posts component 1`] = `
Array [
  <Post
    date="12-12-2020"
    key="12-12-2020"
    text="12"
    url="12"
  />,
  <Post
    date="12-22-2020"
    key="12-22-2020"
    text="12"
    url="12"
  />,
]
`;
